.github/workflows/new-app-review/validators/boilerplate.ts

```diff
@@ -0,0 +1,32 @@
+import type { PullRequestFile, ValidatorOptions, ValidatorResult } from '../../types';
+
+
+export const validate = async (_options: ValidatorOptions, newAppDir: string, files: PullRequestFile[]): Promise<ValidatorResult> => {
+  const hasInvalidBoilerPlate = files
+    .filter((file) => file.status === 'added' && (file.filename.endsWith('.ts') || file.filename.endsWith('.tsx')))
+    .some((file) => {
+
+      if (file.patch) {
+        const paragraphComponentRegex = /<Paragraph[^>]*>([\s\S]*?)<\/Paragraph>/i;
+        const paragraphComponent = file.patch.match(paragraphComponentRegex);
+
+        if (paragraphComponent) {
+          const innerContent = paragraphComponent[1];
+          const helloRegex = /Hello/i;
+          return helloRegex.test(innerContent);
```

Comment:
There might a small potential this could cause a false positive, for example if the partner's app included `<Paragraph>Hello {user}</Paragraph>`

The regex could be expanded to avoid that, although this is probably a small chance of ever having a false positive

```ts
const helloRegex = /Hello \w.+ Component/i;
```